infinispan-spring-boot-starter-remote/src/test/java/test/org/infinispan/spring/starter/remote/container/InfinispanContainer.java

```diff
@@ -0,0 +1,55 @@
+package test.org.infinispan.spring.starter.remote.container;
+
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ClientIntelligence;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.testcontainers.containers.GenericContainer;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+
+@SuppressWarnings("unchecked")
+public class InfinispanContainer extends GenericContainer {
```

Comment:
FYI unchecked warning can be removed if you use `extends GenericContainer<InfinispanContainer>`

Reply:
thanks @bsideup ! 😃